templates/aws-stack.yml

```
@@ -93,6 +94,11 @@ Parameters:
       - edge
     Default: "stable"
 
+  BuildkiteOrgSlug:
+    Description: Buildkite organization slug
```

This might be unclear as to why it's needed, or what it does? It doesn't need to actually match your Buildkite org slug does it, it just scopes the metrics? It'd be so nice if we could just extract it from the agent token somehow 🤔

---

Would it be better to have it blank, and optional, rather than the string "none"?

```suggestion
    Description: Optional Buildkite organization slug (e.g. "my-org"). This only needs to be set when running stacks for different Buildkite organisations in a single AWS account.
```

---

I couldn't think of an easy way to set up the conditionals everywhere else to support an empty one. I guess we could just publish an `Org=` dimension, but that seemed weirder.

---

Where do people see the dimension?